testing/endtoend/params/params.go

```diff
@@ -149,6 +153,8 @@ const (
 	ValidatorMetricsPort = ValidatorGatewayPort + portSpan
 
 	JaegerTracingPort = 9150
+
+	StartupBufferSecs = 5
```

Comment:
TODO: try to run without this, I added it during debugging and can't remember if we need it.